examples/tcpserver/client/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"net/netip"
+	"time"
+)
+
+func main() {
+	const server = "10.0.0.122:1234"
```

Comment:
Maybe add a little comment here to let the user know they have to edit this to match the IP and port the device is listening on.